alpha/declcfg/load.go

```diff
@@ -168,3 +168,16 @@ func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
 	}
 	return cfg, nil
 }
+
+func LoadFile(root fs.FS, file io.Reader, path string) (*DeclarativeConfig, error) {
```

Comment:
Add a GoDoc comment, which should take the form: `<name of symbol> <description of symbol>`, so:

```go 
// LoadFile <... what LoadFile does, what are its inputs/outputs (if not obvious from the types), any interesting behaviors or side effects, etc. ...> 
```